core/iwasm/aot/aot_runtime.h

```diff
@@ -362,7 +374,9 @@ typedef struct AOTFrame {
      *  local area: parameters and local variables
      *  stack area: wasm operand stack
      *  frame ref flags (GC only):
-     *      whether each cell in local and stack area is a GC obj
+     *      whether each cell in local(LLVM JIT) and stack area is a GC obj
+     *                           paddings(AOT)
+     *                           actual ref flags for local are in AOT module
```

Comment:
Here AOTFrame is only for AOT, JIT's frame is WASMInterpFrame.
How about below like:
```C
     *  frame ref flags (GC only):
     *      whether each cell in local and stack area is a GC obj
     *      currently local's ref flags are stored in AOTModule,
     *      here we only reserve the padding bytes 
```